akka-remote/src/main/resources/reference.conf

```diff
@@ -209,7 +209,12 @@ akka {
         inject-handshake-interval = 1 second
 
         # messages that are not accepted by Aeron are dropped after retrying for this period
-        give-up-send-after = 60 seconds
+        give-up-message-after = 60 seconds
+        
+        # System messages that are not acknowledged after re-sending for this period are
+        # dropped and will trigger quarantine. The value should be longer than the length
+        # of a network partition that you need to survive.
+        give-up-system-message-after = 6 hours
```

Comment:
yeah, there are many reasons. We should probably have a section in the docs about quarantine.